src/base/global.cpp

```diff
@@ -127,15 +127,6 @@ void thread_complete()
     app()->thread_complete();
 }
 
-std::string gitCommit()
-{
-#ifdef GIT_COMMIT
-    return GIT_COMMIT;
-#else
-    return "unknown";
-#endif
-}
-
```

Comment:
Why not keep this function around? It can still use the macro from `config.h`, but keeping this would save a lot of the other churn here, I think.

Reply:
The reason for eliminating this was that both `CANTERA_VERSION` and `CANTERA_GIT_COMMIT` are handled the same. It probably should be deprecated though ...